parlai/tasks/huggingface/agents.py

```diff
@@ -39,6 +40,7 @@ def __init__(self, opt, shared=None):
         self.hf_split = self.hf_splits_mapping[self.fold]
         self.data_path = self._path(opt)
         opt['datafile'] = self.data_path
+        make_dir(opt['datafile'])
```

Comment:
Added this so the .lengths files would be saved properly- can leave this here or for consistency do we want to move this into a new build.py file?

Reply:
Huh wouldn't think it was necessary... okay this is fine.